config/root/clusterrolebinding-workspace-home-create.yaml

```diff
@@ -6,4 +6,7 @@ roleRef:
   apiGroup: rbac.authorization.k8s.io
   kind: ClusterRole
   name: system:kcp:home-clusterworkspace-create
-subjects: HOME_CREATOR_GROUPS
+subjects:
+- apiGroup: rbac.authorization.k8s.io
+  kind: Group
+  name: system:authenticated
```

Comment:
wondering, did we really want to remove this? With the change everybody can have a home workspace, right?

Reply:
I only did it because `--home-workspaces-home-creator-groups` is marked deprecated. I had the same question as you. Perhaps we need to un-deprecate it?

Reply:
Restored in latest push